packages/cli/src/utils/checkMissingAppInfo.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async function (app: Object) {
```

Comment:
I'm seeing an error on command line: ![](https://cdn.zappy.app/432e26ea2075f5ce3dbd537c047e89d4.png)

We're not using TypeScript for this project and I don't think node has built-in type annotations for function parameters. Can you please remove the `: Object` in `(app: Object)`?